tests/vm_console_proxy_test.go

```diff
@@ -148,8 +144,10 @@ var _ = Describe("VM Console Proxy Operand", func() {
 		strategy.RevertToOriginalSspCr()
 
 		// Similar workaround as in BeforeEach().
```

Comment:
This workaround is not longer needed, because the feature gate is in `.Spec`, and its update will cause `.metadata.generation` to be updated. So `waitUntilDeployed()` will work as expected.

Can you also remove the workaround from `BeforeEach()`?